recovery/elliptics_recovery/utils/misc.py

```diff
@@ -21,6 +21,7 @@
 import hashlib
 import logging as log
 import msgpack
+import multiprocessing.dummy
```

Comment:
Why is `multiprocessing.dummy.Pool` used, not `multiprocessing.pool.ThreadPool`?